### PR TITLE
Zijie lint fix for reports component Phase 12

### PR DESCRIPTION
**Pull request by iTvX**
# Description
Lint fix for Reports component, this is phase 12. This pull request only fixes two files. PeopleReport.jsx is a big file, including 700 lines of code. I think I should create another pull request for that.
## Files changed
src/components/Reports/PeopleReport/index.js
src/components/Reports/PeopleReport/selectors.js

eslint test:
CleanShot 2023-10-05 at 02 05 09@2x

# How to Test

To test this pull request:

- Check out the current branch
- Run `npm install` and `npm run start:local` to run this PR locally  
- Log in as an admin user
- Check all the functionality of the report is the same as before, and see if any bugs have appeared. (Please record a video)

(Optional)
- Set up ESLint in VSCode/ IntelliJ IDEA, I personally use IntelliJ IDEA
- Delete .eslintignore file in the root: `rm .eslintignore`
- `npx eslint src/components/Reports/PeopleReport/selectors.js`
- `npx eslint src/components/Reports/PeopleReport/selectors.js`
- `git restore .`

I have complete confidence that **the code will pass the ESLint tests.** Therefore, I kindly request that you **focus on testing the frontend**. Specifically, please ensure that the **report functionality remains consistent** with how it operated previously. Thank you for your time and effort!

### CODE REVIEW

**Review by lacnoskillz — APPROVED** [review]
Hi Zijie,

It appears your changes work here as well. Below is a video of me testing. Note that it takes longer to load the total team report data like the other PR's.
I log in as an admin
and test the functionality of the reports page

[Untitled_ Oct 6, 2023 1_08 PM.webm](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114305309/3ff9a515-77a5-4f83-954c-79cefc8789c6)

**Comment by RheaWu1212** [comment]
it works as expected, only thing is: seems like it takes long time after I click 'hide total team report':


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/109626741/99aafd57-cba0-4490-b50e-1e9570df1855

**Review by TuanDinh140194 — DISMISSED** [review]
Hi! I have test your PR with my admin account. For the report function, it works as before. I have go thourgh all the button and search field. It seem work great. However, when I clicked on the show total team report, it load longer than expected. Below is my test PR video. 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/81892917/1f72095b-850a-4f9f-963c-00153a48621c

**Review by Jiyuanxia — APPROVED** [review]
Hi @iTvX , I have reviewed your pull request and tested it. It works as you intended. Well done!


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92963742/4bfbff24-e144-4fba-a122-7a3e535db0ba

**Review by Iris-Song — APPROVED** [review]
Hi! I have test your PR with my admin account. It work as expect. Good job!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/58033867/84a2252a-0465-4c48-acc8-ef37eefd85ac

**Review by Shubhu0 — APPROVED** [review]
Hi @iTvX The reviewed the PR and it is working as Intended

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/56580102/f5df990b-5556-49f9-bc02-cef0aebc7b8f

**Comment by palakgosalia** [comment]
Hi, I have gone through the PR and have tested out the report section, looks good. 


https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104417272/2b2351ea-c914-4210-944a-893b2d402214

**Review by savio-henrique — APPROVED** [review]
Hi @iTvX!

I've tested your PR, everything seems to be working as intended, no errors on the console that are related to this PR, the ESLint passed without any issues and the main functionalities seemed to be working as usual!

Great work at this PR, as usual! :+1: 

--- 

## Video and Screenshot

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/83360232/499f6ded-fac6-48d4-bc90-1bf91c5ff633

terminal

**Review by yi-lin-1234 — DISMISSED** [review]
Hello, functionality of the PeopleReport pages is the same as before. And, the ESLint passed without error. Great work!

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/92185695/9b7ce541-0bc6-4010-bb63-d723df77fcd2

Screenshot 2023-10-31 at 2 46 58 PM

**Review by malikjahanzaib — APPROVED** [review]
Hi @iTvX, I have tested this PR locally and it seems to work as intended without any lint errors. Nice job!!!

Screenshot:

image

Test rec:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46221334/48512a57-32f5-4df7-a7f7-33e16120cbf0

File: src/components/Reports/PeopleReport/index.js [hunk 1]
```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line import/prefer-default-export
```
Comment (EvianTan) [inline src/components/Reports/PeopleReport/index.js:1]: Please fix it instead of disabling it.

**Review by Alforoan — DISMISSED** [review]
It seems to work as before. Nice!
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118868270/9786e246-e1e6-4d96-a6a1-050a71b6d7d6

**Review by EvianTan — APPROVED** [review]

**Pull request merged**